From 99326905ee092471fb117ffd82b939e300034e0a Mon Sep 17 00:00:00 2001
From: Roger Gammans <rgammans@gammascience.co.uk>
Date: Wed, 12 Jul 2023 21:10:53 +0100
Subject: [PATCH] Add alternate name for MX Ergo as found on some devices

Fixes #28349

(cherry picked from commit 681295c0600dc47dbd72cf3fe103b827b24f3b1d)

Related: RHEL-5950
---
 hwdb.d/70-mouse.hwdb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hwdb.d/70-mouse.hwdb b/hwdb.d/70-mouse.hwdb
index 159bc78621..46d723a92e 100644
--- a/hwdb.d/70-mouse.hwdb
+++ b/hwdb.d/70-mouse.hwdb
@@ -539,6 +539,7 @@ mouse:bluetooth:v046dpb019:name:MX Master 2S Mouse:*
 mouse:usb:v046dp406f:name:Logitech MX Ergo:*
 mouse:usb:v046dpc52b:name:Logitech Unifying Device. Wireless PID:406f:*
 mouse:bluetooth:v046dpb01d:name:MX Ergo Mouse:*
+mouse:bluetooth:v046dpb01d:name:Logitech MX Ergo Multi-Device Trackball :*
  ID_INPUT_TRACKBALL=1
  MOUSE_DPI=380@125
 
